### PR TITLE
DE-53 implement smoke tests

### PR DESCRIPTION
**Pull request by dmironov-wallarm**
Implement smoke tests into CI

### CODE REVIEW

File: test/smoke/run-smoke-suite.sh [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+WALLARM_API_CA_VERIFY="${WALLARM_API_CA_VERIFY:-true}"
+WALLARM_API_HOST="${WALLARM_API_HOST:-api.wallarm.com}"
+WALLARM_API_PRESET="${WALLARM_API_PRESET:-eu1}"
+NODE_BASE_URL="${NODE_BASE_URL:-http://workload.default.svc.cluster.local}"
+PYTEST_ARGS=$(echo "${PYTEST_ARGS:---allure-features=Node}" | xargs)
+PYTEST_WORKERS="${PYTEST_WORKERS:-10}"
+#TODO We need it here just to don't let test fail. Remove this variable when test will be fixed.
+HOSTNAME_OLD_NODE="smoke-tests-old-node"
+
+NODE_VERSION=$(< "${CURDIR}/AIO_BASE" awk -F'[-.]' '{print $1"."$2"."$3}')
```
Comment (xDmitriev) [inline test/smoke/run-smoke-suite.sh:39]: We need to get node version from here
https://github.com/wallarm/sidecar/blob/main/helm/values.yaml#L186

File: test/smoke/run-smoke-suite.sh [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+echo "AiO Node version: ${NODE_VERSION}"
```
Comment (xDmitriev) [inline test/smoke/run-smoke-suite.sh:40]:
```suggestion
echo "Node version: ${NODE_VERSION}"
```

File: test/smoke/run-smoke-suite.sh [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+}
+
+function get_logs_and_fail() {
+    get_logs
+    extra_debug_logs
+    clean_allure_report
+    exit 1
+}
+
+function get_logs() {
```
Comment (xDmitriev) [inline test/smoke/run-smoke-suite.sh:53]: we need to align this to sidecar deployment or remove these logs

File: test/smoke/run-smoke-suite.sh [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+  echo "Displaying pods state in default namespace..."
+  kubectl get pods
+
+}
+
+declare -a mandatory
+mandatory=(
+  CLIENT_ID
+  USER_TOKEN
+  # SMOKE_REGISTRY_TOKEN
```
Comment (xDmitriev) [inline test/smoke/run-smoke-suite.sh:127]: both 2 lines required here

File: test/smoke/run-smoke-suite.sh (hunk 1 above)
Comment (dmironov-wallarm) [reply test/smoke/run-smoke-suite.sh]: done

File: test/smoke/run-smoke-suite.sh (hunk 2 above)
Comment (dmironov-wallarm) [reply test/smoke/run-smoke-suite.sh]: done

File: test/smoke/run-smoke-suite.sh (hunk 4 above)
Comment (dmironov-wallarm) [reply test/smoke/run-smoke-suite.sh]: done

File: .github/workflows/ci.yaml [hunk 5]
```diff
@@ -71,3 +72,76 @@ jobs:
...
       load_artifact: true
+
+  smoke-test:
+    name: Smoke
+    runs-on: ${{ matrix.RUNNER }}
+    needs: build
+    strategy:
+      fail-fast: false
+      matrix:
+        k8s: [v1.23.13, v1.27.1]
```
Comment (denisqsound) [inline .github/workflows/ci.yaml:83]: disable pls 1.23.13

File: test/smoke/run-smoke-suite.sh [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+
+# Allure related variables
+ALLURE_ENDPOINT="${ALLURE_ENDPOINT:-https://allure.wallarm.com}"
+ALLURE_PROJECT_ID=${ALLURE_PROJECT_ID:-10}
+ALLURE_RESULTS="${ALLURE_RESULTS:-/tests/_out/allure_report}"
+ALLURE_UPLOAD_REPORT="${ALLURE_UPLOAD_REPORT:-false}"
+ALLURE_GENERATE_REPORT="${ALLURE_GENERATE_REPORT:-false}"
+
+# Pytest related variables
+CLIENT_ID="${CLIENT_ID:-5}"
```
Comment (denisqsound) [inline test/smoke/run-smoke-suite.sh:29]: We need change tests to US cloud

Comment (denisqsound) [reply test/smoke/run-smoke-suite.sh:29]: like in this : https://github.com/wallarm/docker-wallarm-node/pull/85/files

File: test/smoke/run.sh [hunk 7]
```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
```
Comment (xDmitriev) [inline test/smoke/run.sh:3]: please remove this text)

**Review by xDmitriev — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 5 above)
Comment (dmironov-wallarm) [reply .github/workflows/ci.yaml]: done

File: test/smoke/run-smoke-suite.sh (hunk 6 above)
Comment (dmironov-wallarm) [reply test/smoke/run-smoke-suite.sh:29]: done

**Review by denisqsound — APPROVED** [review]

**Pull request merged**